src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,81 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Gripper extends LoggedSubsystem<GripperLoggedInputs> {
+    private final Solenoid solenoid1 = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Gripper.SOLENOID_1);
+    private final Solenoid solenoid2 = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Gripper.SOLENOID_2);
```

Comment:
There is only one solenoid controlling both pistons.

Reply:
I know

Reply:
It sucks

Reply:
done.